core/unix/coredump.c

```diff
@@ -285,6 +285,7 @@ mcontext_to_user_regs(DR_PARAM_IN priv_mcontext_t *mcontext,
     regs->regs[30] = mcontext->r30;
     regs->sp = mcontext->sp;
     regs->pc = (uint64_t)mcontext->pc;
+    regs->pstate = mcontext->nzcv;
```

Comment:
Doesn't the aarch64 term "pstate" include a bunch of other things and not just the NZCV flags?  Are you sure this `pstate` field includes only NZCV?  Please add a comment about this so it's clear.